plugins/Marketplace/Manager.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) 2021 Ultimaker B.V.
+#  Cura is released under the terms of the LGPLv3 or higher.
+from typing import Optional
+
+from PyQt5.QtCore import QObject, pyqtProperty, pyqtSignal
+
+from cura.CuraApplication import CuraApplication
+from UM.PluginRegistry import PluginRegistry
+
+class Manager(QObject):
+    def __init__(self, parent: Optional[QObject] = None):
+        super().__init__(parent = parent)
+        self._manager: "CuraPackageManager" = CuraApplication.getInstance().getPackageManager()
```

Comment:
A manager that has a manager? I'm not really a fan. Could we come up with a better name for this class?

Reply:
Already renamed